torch/lib/THCUNN/SpatialDepthwiseConvolution.cu

```diff
@@ -155,7 +189,7 @@ __global__ void spatialDepthwiseConvolutionAccGradParameters(
   int bidx = blockIdx.x;
   int kW = bidx % kernelWidth;
   int kH = (bidx / kernelWidth) % kernelHeight;
-  int ch = (bidx / channelStride) % kernelChannels;
+  int ch = (bidx / channelStride);
```

Comment:
What happened here? Was the mod unnecessary?

Reply:
Yes. Number of blocks is kernelWidth*kernelHeight* kernelChannels, so blockIdx.x/(kernelWidth*kernelHeight) is less than kernelChannels, thus modulo is unnecessary.